### PR TITLE
Docs: Update whats-new-next with RBAC permission validation

### PR DESCRIPTION
**Pull request by mgyongyosi**
**What is this feature?**

Update Cloud What's new document with permission validation.

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:136]: The first is just a note for me. We'd like to add a release date to these entries for the Cloud version. Can you add that in and then commit this change?
```suggestion
<!-- already in on-prem -->

<Date>

```

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:137]: I might have misunderstood your message in Slack, but we do need this is with an Enterprise license? Or did you mean that's who's already got it but it will be generally available on Cloud?

**Review by imatwawana — COMMENTED** [review]
Thanks for adding this! I've made one small tweak and left a couple questions.

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:139]: @mgyongyosi - Please don't merge this until @jdbaldry has had a chance to review. I'm not sure why the version interpolation syntax isn't working in the link.

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:139]:
```suggestion
With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
```

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (imatwawana) [reply docs/sources/whatsnew/whats-new-next/index.md]: Version issue is resolved!

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (mgyongyosi) [reply docs/sources/whatsnew/whats-new-next/index.md:140]: Apologies, there was a change few months ago that made RBAC available for all of the Cloud licenses, so it's available for all Cloud customers.

File: docs/sources/whatsnew/whats-new-next/index.md
Comment (mgyongyosi) [reply docs/sources/whatsnew/whats-new-next/index.md:150]: I don't know the exact release date, because it started to roll out to different release channels after I merged it (https://github.com/grafana/grafana/pull/73804), so I put the PR's merge date.

**Review by imatwawana — APPROVED** [review]

**Pull request merged**
